policyengine_us/parameters/gov/states/sc/tax/income/subtractions/retirement/subtract_military.yaml

```diff
@@ -0,0 +1,20 @@
+description: South Carolina subtracts the survivors retirement deduction from the military retirement deduction if this is true.
+metadata:
+  unit: bool
+  label: South Carolina subtracts the survivors retirement deduction
+  reference: 
+  - title: Instructions for Form SC-1040 (line 3p)
+    href: https://dor.sc.gov/forms-site/Forms/IITPacket_2021.pdf#page=17
+  - title: Instructions for Form SC-1040 (line 3p)
+    href: https://dor.sc.gov/forms-site/Forms/IITPacket_2022.pdf#page=19
+  - title: SC-1040 2021
+    href: https://dor.sc.gov/forms-site/Forms/SC1040_2021.pdf#page=2
+  - title: SC-1040 2022
+    href: https://dor.sc.gov/forms-site/Forms/SC1040_2022.pdf#page=2
+  # The law indicates the 2022 case, the way of calculation for survivros retirement deduction in 2021 is based on the tax form. 
```

Comment:
Not really sure this is what you're trying to say but please clarify (ChatGPT may be able to help)
```suggestion
  # The law indicates the 2022 case. We model calculation of the survivors retirement deduction in 2021 based on the tax form.
```